src/ui/shared/requests/useQuotes.ts

```diff
@@ -34,86 +49,88 @@ export interface QuotesData {
 
 export function useQuotes({
   address,
-  swapView,
-}: {
-  address: string;
-  swapView: SwapFormView;
-}): QuotesData {
+  slippage,
```

Comment:
```suggestion
  slippage: userSlippage,
```

Reply:
I renamed to `userSlippage` to be more explicit about its usage/intention